packages/nodes-base/nodes/ReadBinaryFile/ReadBinaryFile.node.ts

```diff
@@ -67,6 +71,8 @@ export class ReadBinaryFile implements INodeType {
 				}
 
 				const filePath = this.getNodeParameter('filePath', itemIndex);
+				checkFilePathAccess(filePath);
```

Comment:
The idea is that these variables should be user-defined. 
By making them utilities, we can use them only in specific cases like "readBinary" and "writeBinary," where user input is expected. 
This approach keeps the internal logic intact. Otherwise, we might eventually need to find ways to bypass this added restriction in future development.